src/vtpty/CMakeLists.txt

```diff
@@ -86,3 +86,54 @@ target_include_directories(vtpty PUBLIC
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/include>
 )
 target_link_libraries(vtpty PUBLIC ${vtpty_LIBRARIES})
+
+if(WIN32)
+    option(CONTOUR_EMBED_OPENCONSOLE "Embed OpenConsole" ON)
+    if(NOT CONTOUR_EMBED_OPENCONSOLE)
+        message(STATUS "[vtpty] OpenConsole embedding disabled")
+    else()
+        message(STATUS "[vtpty] OpenConsole embedding enabled")
+        set(OPENCONSOLE_VERSION "1.24.3504.0")
+        set(OPENCONSOLE_VERSION_ALT "1.24.251216004-preview")
+        #set(OPENCONSOLE_VERSION "1.23.13503.0")
+        #set(OPENCONSOLE_VERSION_ALT "1.23.251216003")
+        set(OPENCONSOLE_URL "https://github.com/microsoft/terminal/releases/download/v${OPENCONSOLE_VERSION}/Microsoft.Windows.Console.ConPTY.${OPENCONSOLE_VERSION_ALT}.nupkg")
+        set(OPENCONSOLE_DEST "${CMAKE_BINARY_DIR}/_deps/OpenConsole-${OPENCONSOLE_VERSION}")
+        set(OPENCONSOLE_ZIP "${OPENCONSOLE_DEST}/OpenConsole.zip")
+
+        set(arch "x64")
+        if(CMAKE_SYSTEM_PROCESSOR MATCHES "ARM")
+            set(arch "arm")
+        endif()
```

Comment:
Hm, maybe we can create global scope variable for the arch and use it here?

Reply:
i should therefore chose a better variable name to reflect that :-D

Reply:
renamed